docs/ReleaseNotes.md

```diff
@@ -1,6 +1,77 @@
 # Acuminator Release Notes
 This document provides information about fixes, enhancements, and key features that are available in Acuminator.
 
+## Acuminator 3.1
+Acuminator 3.1 includes the bug fixes and enhancements described in this section, as well as the features that have been implemented in previous versions.
+
+### Enhancements
+In Acuminator 3.1, the following enhancements have been implemented:
+ - The [PX1062](diagnostics/PX1062.md) diagnostic has been added to check that graphs and graph extensions do not contain static actions, views or other static mutable fields and properties.
+ - The following new diagnostics have been added to prevent errors related to the serialization and deserialization of exceptions: 
+   - The [PX1063](diagnostics/PX1063.md) diagnostic checks that the exception classes derived from `System.Exception` always declare a serialization constructor.
+   - The [PX1064](diagnostics/PX1064.md) diagnostic checks that the exception classes derived from `System.Exception` which declare new serializable data, always provide an override of the `GetObjectData` method.
+ - Support for the following complex expressions passed to SetProcessDelegate API has been added to the [PX1008](diagnostics/PX1008.md) and [PX1086](diagnostics/PX1086.md) diagnostics:
+   - Cast expression as shown in the fllowing example:
+     ```C#
+	 ProcessProductSync.SetProcessDelegate((PXProcessingBase<KNCASyncDetails>.ProcessListDelegate)delegate
+	 {
+	    ProductCreateUpdate();
+	 });
+	 ```
+   - Object creation expression as shown in the following example:
+     ```C#
+	 ProcessProductSync.SetProcessDelegate(new PXProcessingBase<KNCASyncDetails>.ProcessListDelegate(delegate
+	 {
+		ProductCreateUpdate();
+	 }));
+	 ```
+   - Local variable or parameter that store delegate
+   - Property or field that store delegate
+ - The [PX1056](diagnostics/PX1056.md) diagnostic has been added to check and warn if a graph is created in the `IsActive` method in graph and DAC extension and the `IsActiveForGraph` method.
```

Comment:
These sounds somewhat strange imho because the last method is just added separately. Maybe we can reword it?
```suggestion
 - The [PX1056](diagnostics/PX1056.md) diagnostic has been added to check and warn if a graph is created in either  `IsActive` or `IsActiveForGraph` methods in a graph extension or DAC extension.
```